managed/ui/src/components/config/Security/certificates/AddCertificateForm.scss

```diff
@@ -0,0 +1,23 @@
+.suggestion {
+  position: relative;
+  top: -53px;
+  width: 100%;
+  height: 42px;
+  padding: 12px 13px;
```

Comment:
make padding 12px for all , as it shows like shadow for text in textbox .
`padding:12px;`

Reply:
Done